utils/testrunner/Matchers.js

```diff
@@ -26,99 +30,202 @@ module.exports = class Matchers {
     this._matchers[name] = matcher;
   }
 
-  expect(value) {
-    return new Expect(value, this._matchers);
+  expect(received) {
+    return new Expect(received, this._matchers);
   }
 };
 
+class MatchError extends Error {
+  constructor(message, formatter = null) {
```

Comment:
nit: formatter = defaultFormatter
Then you can avoid a null check later.

Reply:
formatter is bound at least to the `received` argument; but the `= null` part is actually never needed. Thanks!